.github/workflows/ci.yml

```diff
@@ -54,9 +54,14 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Use Rust ${{ matrix.rust-toolchain }}
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@v1
         with:
           toolchain: ${{ matrix.rust-toolchain }}
+          components: clippy,rustfmt
+
+      - name: Linkme nightly workaround
+        if: matrix.rust-toolchain == 'nightly' && matrix.os == 'ubuntu-latest'
+        run: echo RUSTFLAGS=${RUSTFLAGS}\ -Clink-arg=-Wl,-z,nostart-stop-gc >> $GITHUB_ENV
```

Comment:
Should leave a comment linking to https://blog.rust-lang.org/2024/05/17/enabling-rust-lld-on-linux.html#possible-drawbacks